app/scripts/modules/core/src/pipeline/config/triggers/artifacts/artifact.component.ts

```diff
@@ -52,7 +52,7 @@ class ArtifactCtrl implements IController {
 
     if (artifactKindConfig.length) {
       const config = artifactKindConfig[0];
-      const template: Element = config.template as any;
+      const template: string = config.template as any;
```

Comment:
Isn't the type of `template` on `IArtifactKindConfig` a `string`? Should be able to drop `as any`.